src/chrome/content/rules/IronRuby.net.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="IronRuby.net">
+	<target host="ironruby.net" />
```

Comment:
Oops, just ignore my previous comment.
See also `www.ironruby.net` is what i meant. Since it does not have a valid cert you should add a comment about this issue and redirect it to `https://ironruby.net/`.